products/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+from users.models import TimeStampedModel
+
+class Product(TimeStampedModel):
+    product_name    = models.CharField(max_length=15)
+    product_content = models.CharField(max_length=300)
```

Comment:
Product class의 member 변수로 name, content가 선언되기 때문에 앞에 `product_`를 붙이지 않아도 될 것 같습니다!

```suggestion
    name    = models.CharField(max_length=15)
    content = models.CharField(max_length=300)
```